sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -845,6 +845,25 @@ static const int ImmediateCommandlistsSetting = [] {
   return std::stoi(ImmediateCommandlistsSettingStr);
 }();
 
+// Get value of the threshold for number of events in immediate command lists.
+// If number of events in the immediate command list exceeds this threshold then
+// cleanup process for those events is executed.
+static const size_t ImmCmdListsEventCleanupThreshold = [] {
+  const char *ImmCmdListsEventCleanupThresholdStr = std::getenv(
+      "SYCL_PI_LEVEL_ZERO_IMMEDIATE_COMMANDLISTS_EVENT_CLEANUP_THRESHOLD");
+  static constexpr int Default = 20;
+  if (!ImmCmdListsEventCleanupThresholdStr)
+    return Default;
+
+  int Threshold = std::stoi(ImmCmdListsEventCleanupThresholdStr);
```

Comment:
atoi doesn't throw

```suggestion
  int Threshold = std::atoi(ImmCmdListsEventCleanupThresholdStr);
```

Reply:
I can change, for some reasons we have mix of both in the plugin.

Reply:
feel free to fix other uses of "stoi", change to "atoi" or try-catch

Reply:
Fixed to atoi.